### ISSUE TITLE
CCP Tasklist Queue Clearing

### ISSUE THREAD

**Issue by msweier** [opendcs/opendcs#800]
**Is your feature request related to a problem? Please describe.**
A simple tool that will clear the queue for a runaway process. This ties in with #799.

**Describe the solution you'd like**
It could run this SQL with the process ID as an argument:
`Delete from cp_comp_tasklist where loading_application_id in (processId)`

**Additional context**
Ideally integrated in with #799 with a GUI.

**Comment by MikeNeilson** [comment]
Unfortunately this is easier said that done. One of the major problems with the task list that makes things slow, would the operation here also wouldn't work.

that said a not to the user to just turn everything off would probably help with that.

**Comment by MikeNeilson** [comment]
sort of, it's just challenging due factors external to the database.

as for #743, SPK is having a similar issue due to an otherwise reasonable backload.

I'm for adding this to an interface, just need to manage expectations given the current database usage.

**Comment by msweier** [comment]
Yeah the GUI bit is just so we can monitor once we are in the cloud. I assume we won't be able run any commands in there at the district level.

**Comment by MikeNeilson** [comment]
Given some of the changes we'd like to make for cloud we'll probably have other methods. But there does at least need to be some sort of standard interface (in the code) to the tasklist, and we'll likely just start with a commandline problem - it's just easier.
